core/objects.go

```diff
@@ -172,16 +172,16 @@ type Registration struct {
 // and the IP addresses that were resolved and used
 type ValidationRecord struct {
 	// DNS only
-	Authorities []string `json:",omitempty"`
+	Authorities []string `json:"-"`
```

Comment:
This means that this information is now only stored in logs and is no longer kept in the DB because of how we store the blob as JSON, is that what we want?

Reply:
I was under the impression this field was basically useless in practice. I think that's having it in-log only is fine?

Reply:
I think we should go a little deeper and remove the field from ValidationRecord. Filed a follow up ticket: #3407.